addons/survey/views/survey_user_views.xml

```diff
@@ -67,9 +67,9 @@
                             <field name="is_attempts_limited" invisible="1"/>
                             <label for="attempts_number" string="Attempt n°" attrs="{'invisible': ['|', ('is_attempts_limited', '=', False), '|', ('test_entry', '=', True), ('state', '!=', 'done')]}"/>
                             <div attrs="{'invisible': ['|', ('is_attempts_limited', '=', False), '|', ('test_entry', '=', True), ('state', '!=', 'done')]}">
```

Comment:
```suggestion
                            <div attrs="{'invisible': ['|', ('is_attempts_limited', '=', False), '|', ('test_entry', '=', True), ('state', '!=', 'done')]}"  class="d-inline-flex">
```
I would just add the class on the parent div :wink: 

Why not do the same with the div containing `scoring_percentage` to make sure the `%` sign remains on the same line ?

Reply:
@epictete thanks for showing the better way to achieve this, updated for scoring % field also